app/src/main/java/org/oppia/android/app/settings/profile/ProfileResetPinViewModel.kt

```diff
@@ -8,21 +8,21 @@ import javax.inject.Inject
 /** The ViewModel for [ProfileResetPinActivity]. */
 @ActivityScope
 class ProfileResetPinViewModel @Inject constructor() : ObservableViewModel() {
-  /** Variable for storing error message in first text input layout. */
+  /** The error message is shown if the second pin not matches with the first pin. */
   val pinErrorMsg = ObservableField("")
 
-  /** Variable for storing error message in second text input layout. */
+  /** The error message is confirmed with this pin. */
   val confirmErrorMsg = ObservableField("")
 
-  /** For confirming if the profile is admin or not. */
+  /** If the user is admin then he should have admin rights. */
```

Comment:
Isn't this more like: 'Whether the user is an admin'?

Reply:
done